packages/cli/src/commands/sandbox/sandbox_command.ts

```diff
@@ -201,4 +196,16 @@ export class SandboxCommand
         await this.sandboxFactory.getInstance()
       ).delete({ name: this.sandboxName });
   };
+
+  validateDirectory = async (option: string, dir: string) => {
```

Comment:
This should be private

Reply:
Done in [79acaf7](https://github.com/aws-amplify/amplify-backend/pull/643/commits/79acaf77661c20561726831e20a7fb657fb677f0)